tsl/src/hypercore/hypercore_handler.c

```diff
@@ -3372,21 +3445,13 @@ convert_to_hypercore(Oid relid)
 		/* A compressed relation already exists, so converting from legacy
 		 * compression. It is only necessary to create the proxy vacuum
 		 * index. */
-		create_proxy_vacuum_index(relation, hsinfo->compressed_relid);
+		create_proxy_vacuum_index(relation, hcinfo->compressed_relid);
 		table_close(relation, AccessShareLock);
 		return;
```

Comment:
Just an observation. This do not have to do with your change, but we should have a test for this part of the code.

Reply:
Sure, if possible.